channels/it.m3u

```diff
@@ -336,24 +332,18 @@ https://mediapolis.rai.it/relinker/relinkerServlet.htm?cont=2606803
 http://mediapolisevent.rai.it/relinker/relinkerServlet.htm?cont=2606803
 #EXTINF:-1 tvg-id="Rai2.it" tvg-name="Rai 2" tvg-country="IT" tvg-language="" tvg-logo="" group-title="",Rai 2 [Geo-blocked]
 https://mediapolis.rai.it/relinker/relinkerServlet.htm?cont=308718
-#EXTINF:-1 tvg-id="Rai2Geo.it" tvg-name="Rai 2 (Geo)" tvg-country="IT" tvg-language="" tvg-logo="" group-title="",Rai 2 (Geo)
-http://mediapolisevent.rai.it/relinker/relinkerServlet.htm?cont=180116
```

Comment:
Should work, but the Geo-blocked mention was wrongly written.

Reply:
This link doesn't work for me even via VPN.

Reply:
Indeed. Not found on my side.